Providers/Modules/Plugins/ChangeTracking/conf/change_tracking_inventory.mof

```diff
@@ -0,0 +1,38 @@
+/*
+@TargetNode='Localhost'
+*/
+
+instance of MSFT_nxPackageResource
+{
+                Name = "*";
+                ResourceId = "[MSFT_nxPackageResource]Inventory";
```

Comment:
I copied the mof file from the omi. I think the indentation is special in the "meat" of the content for attention.